libdevcore/SwarmHash.h

```diff
@@ -20,12 +20,13 @@
 #pragma once
 
 #include <libdevcore/FixedHash.h>
-#include <libdevcore/Common.h>
+
+#include <string>
```

Comment:
nitpick: the coding rule says STL includes should be placed above `libdevcore/*`.